node_agent/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Istio Authors
```

Comment:
+1 I think that's a good thing to do. So the directory structure should be something have the following, right?
```
ca/
   cmd/
   ...
   main.go
   BUILD
node_agent/
   cmd/
   ...
   main.go
   BUILD
util/
BUILD
```